tests/lib/test_Repo.py

```diff
@@ -453,30 +460,27 @@ def test_Repo_opdir_root(tmp_path: str) -> None:
     assert repo.root.samefile(repo.opdir)
 
 
-def test_Repo_opdir_child(tmp_path: str) -> None:
+@pytest.mark.repo_dirs('1/2/3/4/5/6')
+def test_Repo_opdir_child(repo: Repo, tmp_path: Path) -> None:
     """
-    When instantiating the Repo with the cwd being inside a folder, the property
-    repo.opdir must be the cwd.
+    An existing Repo must be instantiated with the root of a repository,
+    otherwise an error is raised.
     """
-    os.chdir(tmp_path)
+    os.chdir(Path(repo.root, '1/2/3/4/5/6'))
```

Comment:
Minor nitpick: Since `repo.root` is already a `Path` instance, you can use `repo.root / '1' / '2' / '3' / '3' / '4' / '5' / '6'. I believe Ben suggested this use with good reason.